lib/rmg/modificators/TreasurePlacer.cpp

```diff
@@ -831,7 +837,7 @@ void TreasurePlacer::createTreasures(ObjectManager& manager)
 	{
 		for (auto* oi : treasurePile)
 		{
-			oi->destroyObject();
+			//oi->destroyObject();
```

Comment:
This one is neccessary, actually likely to happen as RMG fails to find place for treasure pile.

Reply:
Suggestions how to replace? Because at the moment what we have is crash. Which is way worse than too many banned heroes.

Perhaps store created object in `oi`? Can only one object be created using same `oi` instance?

Reply:
Or perhaps that functor should instead be called as part of `Object::clear` / `Object::Instance::clear`?